spec/features/swagger_spec.rb

```diff
@@ -16,6 +16,13 @@
     before do
       @options = GrapeSwaggerRails.options.dup
     end
+
+    it 'evaluates config options correctly' do
+      visit '/swagger'
+      page_options_json = page.evaluate_script("$('html').data('swagger-options')").to_json
+      expect(page_options_json).to eq(@options.marshal_dump.to_json)
+    end
```

Comment:
That one should have, I didn't realize that jquery unescapes html entities when processing a data tag. So there was no bug, except that the contents of the `<html>` tag were difficult to read.